### PR TITLE
Release 0.2.72

### PR DESCRIPTION
**Pull request by johnc1231**
@daniel-goldstein Any changes to batch that should be documented?

### CODE REVIEW

File: hail/python/hail/experimental/table_ndarray_utils.py [hunk 1]
```diff
@@ -18,8 +18,11 @@ def mt_to_table_of_ndarray(entry_expr, block_size=16, return_checkpointed_table_
 
     def get_even_partitioning(ht, partition_size, total_num_rows):
         ht = ht.select().add_index("_even_partitioning_index")
+        info("Added index")
```
Comment (tpoterba) [inline hail/python/hail/experimental/table_ndarray_utils.py:21]: looks like another commit snuck in?

**Review by tpoterba — CHANGES_REQUESTED** [review]

File: hail/python/hail/experimental/table_ndarray_utils.py (hunk 1 above)
Comment (johnc1231) [reply hail/python/hail/experimental/table_ndarray_utils.py:21]: Ugh yes, let me fix.

Comment (johnc1231) [reply hail/python/hail/experimental/table_ndarray_utils.py]: Accidentally forked off a debugging branch, rebased it away.

**Review by tpoterba — APPROVED** [review]

**Comment by tpoterba** [comment]
Tagging WIP until someone from services team comments on that part of the changelog

**Comment by daniel-goldstein** [comment]
#10612 Is the only user-visible change to batch since the last release.
> CHANGELOG: Made failed Python Jobs have non-zero exit codes.

**Pull request merged**